src/materials/MeshPhongMaterial.js

```diff
@@ -11,6 +11,8 @@
  *
  *  map: new THREE.Texture( <Image> ),
  *
+ *  energyConverving: false
```

Comment:
Yeah, that's a good point.

@bhouston do you mind leaving the the `energyConserving` property for another PR? The rest looks great though!